src/sources/http.rs

```diff
@@ -175,10 +175,12 @@ impl HttpSource for SimpleHttpSource {
                 }
                 Ok(None) => break,
                 Err(error) => {
+                    // Error is logged / emitted by `crate::codecs::Decoder`, no further
+                    // handling is needed here
```

Comment:
I was 99% sure I/we updated the internal events for codecs already - or was that just Encoding?

Reply:
It looks like you added them yes, and I made a modification in this PR:

https://github.com/vectordotdev/vector/pull/14021/files#diff-6f76e603a28a915a3113b9166df34b2e9872d285b28515226796b3997d5f88a2

Reply:
😅 lol, yeah my brain is good after a week off.